lib/errors.js

```diff
@@ -72,6 +78,27 @@ Errors.prototype = {
     },
 
     /**
+     * Formats message according to message data and _verbose flag
+     *
+     * @param {String|Error} message
+     */
+    _prepareMessage: function(message) {
+        if (!this._verbose) {
```

Comment:
I think this should happen regardless of verbose flag. Reporting issues is not something that should be dictated by a verbose flag or not.